project_task_pull_request/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Project Task Pull Request",
     "summary": "Adds a field for a PR URI to project tasks",
-    "version": "15.0.1.0.1",
+    "version": "16.0.1.0.1",
```

Comment:
16.0.1.0.0

Reply:
hi @pedrobaeza is this the only one issue here? If yes and @valjaro is not available any longer would't you mind us reopening this PR from our own branch so we could have it merged?

Reply:
Yeah, it seems so.